lib/bolt/config.rb

```diff
@@ -251,6 +251,7 @@ def initialize(project, config_data, overrides = {})
         'format'              => 'human',
         'log'                 => { 'console' => {} },
         'module-install'      => {},
+        'modules'             => [],
```

Comment:
```suggestion
```

Reply:
Why? This should be the default.

Reply:
Oh, because it can't be set in `bolt-defaults.yaml`. Hm...ok.